x/distribution/client/cli/tx.go

```diff
@@ -0,0 +1,117 @@
+// nolint
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/client/utils"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/client/txbuilder"
+
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+var (
+	flagOnlyFromValidator = "only-from-validator"
+	flagIsValidator       = "is-validator"
```

Comment:
these flag names are confusing relative to what they do... one takes addr bytes and the other bech32.